### ISSUE TITLE
the predicted result is nan

### ISSUE THREAD

**Issue by kaijieqin** [JihongJu/keras-fcn#6]
HI

I run your program and find that parameters in some layers are **## nan**,  then I run the program with mse, it seems ok. Is there something wrong with the crossentropy?

**Comment by JihongJu** [comment]
Hi, I'm not sure if I understand your problem correctly. By "parameters in some layers are ## nan", I assume you mean the weights in some layers are `nan`. Do you get `nan` losses in that case?

Usually `nan` loss (or inf loss) is caused by improper configuration of hyperparameters, for example, the learning rate is to0 large.

**Comment by kaijieqin** [comment]
Hi, I find that the loss is nan and the parameters after conv5 are nan.
1. where can I set the learning rate?
2.can you upload you trained model?

**Comment by JihongJu** [comment]
Hi, 

1. You can change the arguments of the optimizer while fitting a data generator. See [Keras Optimizers](https://keras.io/optimizers/)
2. Unfortunately, I don't have a trained model. At the moment, I am using a pre-trained model converted from the Caffe model zoo. (See #7)
